lib/Gateway.js

```diff
@@ -1146,11 +1144,8 @@ Gateway.prototype.discoverDevice = function (node, hassDevice) {
       }
     }
   } catch (error) {
-    debug(
-      'Error while discovering device %s of node %d: %s',
-      hassID,
-      node.id,
-      error
+    logger.error(
+      `Error while discovering device ${hassID} of node ${node.id}: ${error}`
```

Comment:
Print also the error stack, this could make easier to undestand where the discovery is broken

Reply:
done.